rll/setup-automatic-upgrade.sh

```diff
@@ -44,28 +50,51 @@ else
     rsc_command="schedule_recipe /api/right_net/scheduler/schedule_recipe recipe=rll::upgrade"
   fi
 
+  # Random hour 0-23
+  scheduled_hour=$(( $RANDOM % 24 ))
+
+  # Random minute 0-59
+  scheduled_minute=$(( $RANDOM % 60 ))
+
+  if [[ $(cat /etc/os-release) =~ CoreOS ]]; then
```

Comment:
Lots of oses don't have the os-release file, in which case the script will error out here. I like using grep here. Its a fairly compact way to trigger only if both the file exists and contains the coreos string:
if grep -qi coreos /etc/os-release >/dev/null 2>&1; then